tap_wikipedia/models/config.py

```diff
@@ -0,0 +1,38 @@
+from pathlib import Path
+from typing import Annotated
+
+from appdirs import user_cache_dir
+from pydantic import BaseModel, Field, field_validator
+
+from tap_wikipedia.models.types import EnrichmentType, SubsetSpecification
+
+
+class Config(BaseModel):
```

Comment:
Nit: this is Singer config but you may still want to use BaseSettings.

Reply:
I have changed this to BaseSettings.